blazor/treeview/check-box.md

```diff
@@ -99,4 +101,646 @@ By default, the checkbox state of parent and child nodes are dependent on each o
+</SfTreeView>
+
+@code {
+    // Specifies the CheckedNodes value for TreeView component.
+    public string[] CheckedNodes = new string[] { "2", "4" };
+    public class MusicAlbum
+    {
+        public int Id { get; set; }
+        public int? ParentId { get; set; }
+        public string? Name { get; set; }
+        public bool Expanded { get; set; }
+        public bool? IsChecked { get; set; }
+        public bool HasChild { get; set; }
+    }
+    List<MusicAlbum> Albums = new List<MusicAlbum>();
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();
+        Albums.Add(new MusicAlbum
+            {
+                Id = 1,
+                Name = "Discover Music",
+                HasChild = true,
+                Expanded = true
+            });
+        Albums.Add(new MusicAlbum
+            {
+                Id = 2,
+                ParentId = 1,
+                Name = "Hot Singles"
+            });
+        Albums.Add(new MusicAlbum
+            {
+                Id = 3,
+                ParentId = 1,
+                Name = "Rising Artists"
+            });
+        Albums.Add(new MusicAlbum
+            {
+                Id = 4,
+                ParentId = 1,
+                Name = "Live Music"
+            });
+        Albums.Add(new MusicAlbum
+            {
+                Id = 14,
+                HasChild = true,
+                Name = "MP3 Albums",
+                Expanded = true
+
+            });
+        Albums.Add(new MusicAlbum
+            {
+                Id = 15,
+                ParentId = 14,
+                Name = "Rock"
+            });
+        Albums.Add(new MusicAlbum
+            {
+                Id = 16,
+                Name = "Gospel",
+                ParentId = 14,
+            });
+        Albums.Add(new MusicAlbum
+            {
+                Id = 17,
+                ParentId = 14,
+                Name = "Latin Music"
+            });
+        Albums.Add(new MusicAlbum
+            {
+                Id = 18,
+                ParentId = 14,
+                Name = "Jazz"
+            });
+    }
+}
+```
+
+## Check and uncheck all nodes
+
+The Blazor TreeView component offers the ability to check all unchecked nodes within the component by utilizing the [CheckAllAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfTreeView-1.html#Syncfusion_Blazor_Navigations_SfTreeView_1_CheckAllAsync_System_String___) method. Additionally, specific nodes can be selected by passing in an array of unchecked nodes. The [UnCheckAllAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfTreeView-1.html#Syncfusion_Blazor_Navigations_SfTreeView_1_UncheckAllAsync_System_String___) method also exists to uncheck all previously checked nodes within the component, and specific nodes can be deselected by passing in an array of checked nodes. 
```

Comment:
Make sure the casing for APIs.

UncheckAllAsync or UnCheckAllAsync?

Reply:
Done.